code/edge/src/main/java/com/adobe/marketing/mobile/NetworkResponseHandler.java

```diff
@@ -31,6 +33,8 @@
  */
 class NetworkResponseHandler {
 
+	private static final String LOG_SOURCE = "NetworkResponseHandler";
```

Comment:
I've only updated NetworkResponseHandler to fix the cases of `Log` which were failing to compile. The rest of this class will be updated in another PR.